app/cdap/components/AbstractWidget/index.tsx

```diff
@@ -44,7 +44,7 @@ export interface IWidgetExtraConfig {
 
 export interface IWidgetProps<T = any> {
   widgetProps?: T;
-  value: string | number;
+  value?: string | number;
```

Comment:
This is not set when the field is empty?

Reply:
this one might've been a mistake let me double check it